backend/aurcache/src/builder/docker.rs

```diff
@@ -178,11 +178,27 @@ impl Builder {
 
         let (makepkg_config, makepkg_config_path) =
             create_makepkg_config(name.clone(), build_dir_base)?;
+
+        // Do we need to update everything (paru -Syu) every time?
+        let init_cmd = "
+            sudo pacman-key --init
+            sudo pacman-key --populate archlinux
+            paru -Syu
+        ";
+        // Do we actually need custom build flags? Since we're not installing anymore, there's not
+        // much we _could_ do.
         let build_cmd = format!(
-            "sudo pacman-key --init && sudo pacman-key --populate archlinux && paru {build_flags} {name}"
+            "cd {build_dir_base}
+             paru -G {name}
+             paru {build_flags} {name}"
         );
+        info!("Init command: {init_cmd}");
         info!("Build command: {build_cmd}");
-        let cmd = format!("cat <<EOF > {makepkg_config_path}\n{makepkg_config}\nEOF\n{build_cmd}");
+        let cmd = format!(
+            "cat <<EOF > {makepkg_config_path}\n{makepkg_config}\nEOF
+            {init_cmd}
+            {build_cmd}"
+        );
```

Comment:
Here I thought it would be more readable to have a multi-line literal rather than escaped `\n`. But for the EOF I kept the escaped newlines, to avoid having indentation messing with it.